mkdocs.yml

```diff
@@ -10,8 +10,8 @@ edit_uri: ""
 
 theme:
   name: 'material'
-  logo: img/Planet_primarymark_RGB_White.png
-  favicon: img/logo.svg
+  logo: 'images/Planet_primarymark_RGB_White.png'
```

Comment:
Note that this logo is not checked into git, and I see:

<img width="373" alt="Screen Shot 2022-11-14 at 11 06 03 AM" src="https://user-images.githubusercontent.com/407017/201744531-4f1cf678-ca7d-434d-8d35-b224faa0e87d.png">

which I'm guessing is where this is supposed to show up?